scripts/paycontract.py

```diff
@@ -0,0 +1,33 @@
+def pay(pay, balance1, balance2):
+    balance1 = int(balance1)
+    balance2 = int(balance2)
+    pay = int(pay)
+    if(pay > balance1):
```

Comment:
spacing after the if is preferable for readability
```suggestion
    if (pay > balance1):
```